tests/api/DeleteCardEndpointCest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Tests;
+
+use App\Tests\ApiTester;
+use App\Entity\Card;
+use Codeception\Util\HttpCode;
+
+class DeleteCardEndpointCest
+{
+    public function _before(ApiTester $I)
+    {
+    }
+
+    public function testItDeleteCard(ApiTester $I)
+    {
+        $I->haveInRepository(
+            $existedCard = new Card(
+                'Goblin',
+                1,
+                1,
+                1,
+            )
+        );
+
+        $I->sendDelete('card/1');
+        $I->seeResponseCodeIs(HttpCode::OK);
+    }
+
+    public function testItThrowsErrorWhenCardNotFound(ApiTester $I)
+    {
+        $I->haveInRepository(
+            $existedCard = new Card(
+                'Goblin',
+                1,
+                1,
+                1,
+            )
+        );
+
+        $I->sendDelete('card/99999');
+        $I->dontSeeResponseCodeIs(HttpCode::OK);
```

Comment:
What if we got System Error here?

Reply:
You mean that it should expect exception rather then looking for NON OK status?

Reply:
Not exactly. I mean we should check exactly whats we want/expect to happen. Because currently if somethings goes wrong and API return 500 then this test will be green. But thats wrong because user expect 404.

Reply:
Not exactly. I mean we should check exactly whats we want/expect to happen. Because currently if somethings goes wrong and API return 500 then this test will be green. But thats wrong because user expect 404.

Reply:
so such approach would be better? $I->SeeResponseCodeIs(HttpCode::NOT_FOUND);